### PR TITLE
Updated ino file with Lora and serial port initialization for control…

### PR DESCRIPTION
**Pull request by AbhinandSuresh**
Please review the code

### CODE REVIEW

File: AWMS_Controller/AWMS_Controller.ino [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+   while (true);                     // if failed, halt.
+  }
+}
+
+//function to initilize serial communication for debugging
+void serialInit()
+{
+  Serial.begin(SERIAL_BAUD);
+}
+void setup()
```
Comment (greatinbaby) [inline AWMS_Controller/AWMS_Controller.ino:57]: space

File: AWMS_Controller/AWMS_Controller.ino [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+//function to initilize serial communication for debugging
+void serialInit()
+{
+  Serial.begin(SERIAL_BAUD);
+}
+void setup() 
+{
+  /*
```
Comment (greatinbaby) [inline AWMS_Controller/AWMS_Controller.ino:59]: remove these comments

File: AWMS_Controller/AWMS_Controller.ino [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+   * add setup and initialization routine(s) here
+  */
+  //Initialize Lora and halt if any failure.
+  serialInit();
```
Comment (greatinbaby) [inline AWMS_Controller/AWMS_Controller.ino:63]: serial init for lora?

File: AWMS_Controller/AWMS_Controller.ino [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  debug_Println(1,"\nSerial Init Success...");
+  loraInit();
+  debug_Println(1,"Lora Init Success...");
+  /*
```
Comment (greatinbaby) [inline AWMS_Controller/AWMS_Controller.ino:67]: remove this

File: AWMS_Controller/AWMS_Controller.ino [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+void serialInit()
+{
+  Serial.begin(SERIAL_BAUD);
+}
+void setup() 
+{
+  /*
+   * add setup and initialization routine(s) here
+  */
+  //Initialize Lora and halt if any failure.
```
Comment (greatinbaby) [inline AWMS_Controller/AWMS_Controller.ino:62]: halt? where

File: AWMS_Controller/AWMS_Controller.ino [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+const int irqPin = 4;           // change for your board; must be a hardware interrupt pin
+
+//function to initialize LoRa
+void loraInit()
+{
+  LoRa.setPins(csPin, resetPin, irqPin);
+  if (!LoRa.begin(LORA_FREQUENCY)) 
+  {
+   debug_Println(1,"\nStarting LoRa failed..! Error.");
+   while (true);                     // if failed, halt.
```
Comment (greatinbaby) [inline AWMS_Controller/AWMS_Controller.ino:48]: loop for ever

File: AWMS_Monitor/AWMS_Monitor.ino [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+   Communication is done using radio wave transmission of packets.
+   LoRa modules are used for transmitting datas.
+*/
+#include <SPI.h>              // include libraries
+#include <LoRa.h>
+#define LORA_FREQUENCY 433E6  // Set LoRa frequency 
+#define SERIAL_BAUD 9600      // Serial baud rate
+#define CS_PIN 10             // LoRa radio chip select
+#define RST_PIN 9             // LoRa radio Reset pin
+#define IRQ_PIN 2             // LoRa hardware interrupt pin
```
Comment (greatinbaby) [inline AWMS_Monitor/AWMS_Monitor.ino:16]: both programs should be same in style , defining pins

File: AWMS_Monitor/AWMS_Monitor.ino [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+#define debug_Print(level,msg) { \
+        if(DEBUG_LEVEL <= level) \
+        Serial.print(msg); \
+        }
+#else
+#define debug_Println(x)
+#define debug_Print(x)
+#endif
+
+void serialInit()
```
Comment (greatinbaby) [inline AWMS_Monitor/AWMS_Monitor.ino:41]: below two functions can be directly added to setup

File: AWMS_Controller/AWMS_Controller.ino (hunk 1 above)
Comment (AbhinandSuresh) [reply AWMS_Controller/AWMS_Controller.ino]: removed space

File: AWMS_Controller/AWMS_Controller.ino (hunk 2 above)
Comment (AbhinandSuresh) [reply AWMS_Controller/AWMS_Controller.ino]: done

File: AWMS_Controller/AWMS_Controller.ino (hunk 4 above)
Comment (AbhinandSuresh) [reply AWMS_Controller/AWMS_Controller.ino]: okey

File: AWMS_Monitor/AWMS_Monitor.ino (hunk 8 above)
Comment (AbhinandSuresh) [reply AWMS_Monitor/AWMS_Monitor.ino]: okey

**Review by greatinbaby — APPROVED** [review]
Approved +1

**Review by peterpathros — APPROVED** [review]

**Pull request merged**